console/ui/src/app/login/login.component.ts

```diff
@@ -44,6 +51,7 @@ export class LoginComponent implements OnInit {
     this.loginForm = this.formBuilder.group({
       username: ['', Validators.compose([Validators.required])],
       password: ['', Validators.compose([Validators.required, Validators.minLength(8)])],
+      code: [{ value: '', disabled: true }, Validators.compose([Validators.required, Validators.minLength(6), Validators.maxLength(6)])],
```

Comment:
Max length 8 for the recovery codes.

```suggestion
      code: [{ value: '', disabled: true }, Validators.compose([Validators.required, Validators.minLength(6), Validators.maxLength(8)])],
```